backend/capellacollab/users/tokens/routes.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import base64
+
+import fastapi
+from sqlalchemy import orm
+
+from capellacollab.core import credentials, database
+from capellacollab.core.authentication import injectables as auth_injectables
+from capellacollab.users import injectables as user_injectables
+from capellacollab.users import models as users_models
+
+from . import crud
+
+router = fastapi.APIRouter(
+    dependencies=[
+        fastapi.Depends(
+            auth_injectables.RoleVerification(
+                required_role=users_models.Role.USER
+            )
+        )
+    ]
+)
+
+
+@router.get("/{user_id}/token")
+def get_token_for_user(
```

Comment:
I would make this a `POST` instead of a `GET` as we actually create something and a `GET` should only be used to retrieve data. Since we don't need anything, the body can simply be empty.